scripts/installscripts/buster-install-default.sh

```diff
@@ -545,6 +545,7 @@ cd /home/pi/RPi-Jukebox-RFID || exit
 
 # Install more required packages
 sudo pip install -r requirements.txt
+sudo pip3 install -r /home/pi/RPi-Jukebox-RFID/components/rfid-reader/PN532/requirements.txt
```

Comment:
@s-martin @MiczFlor at some point we should think about if it is a good idea to install all dependencies for all hardware-combinations we can think of. but for now I'm fine with installing one more.

Reply:
I think that - for the time being - we won't clutter the SD card of the Pi with the libraries. The install process will take a little longer, the response time should make no difference.
So I agree for now.